docs/managing-mods.md

```diff
@@ -0,0 +1,37 @@
+# Managing mods
+
+> Learn how to manage your mods using Lampray.
+
+## Prerequisites
+
+Before you can manage your mods using Lampray, you'll need to complete one of the following:
+
+- [Download the latest release](../README.md#quick-start)
+- [Build from source](./building-from-source.md).
+
+## Step 1: Set your game paths
+
+You'll need to set the path your Steam library and game data in Lampray, so your mods are added to the right directories.
+
+At the top of the window, select **Lampray** > **Steam Directory**, then follow the on-screen instructions to set the path to your Steam game's root directory and `AppData` directory.
+
+## Step 2: Manage your mods
+
+You'll need to complete the following steps anytime you add or remove mods using Lampray:
+
+### Adding mods
+
+> **Note:** At this time, mods using `.rar` files are not supported.
+
+1. Download your mods from a mod repository, such as Nexus Mods.
+2. At the top corner, choose the game you'd like to mod.
+3. Drag and drop the `.zip` packages for each mod into Lampray.
+4. Enable each mod, choose their mod type, then set their load order. For more information about each mod type, see [Mod types](about-lampray/mod-types.md). 
```

Comment:
This currently has a dead link. It might also be good to tweak the wording a bit for now (and change this later if we have a more generic page):
> Enable each mod, choose their mod type, then set their load order. For more information about Baldurs Gate 3 mod types, see (linK)

I am not quite sure I like how that looks either, though. Ideally, I think we would want to additionally mention that some games may not need you to choose mod types, but that can probably be dealt with at a later time.

Reply:
Good catch. Rewriting now. But to your point, this will be a temp solution.

---

### Moving forward

In my experience, if the docs start to feel a bit to complicated, it _may_ be an indication that the software design itself is complicated. My suggestion would be: in the future Lampray should have a set amount of mod types+definitions. For example:

- Lampray has 15 mod types, each with their own unique/specific definition.
- BG3 supports 8 mod types. This is listed in the BG3 doc.
- Cyberpunk supports 1 mod type. This is listed in the Cyberpunk doc.
- Skyrim supports 4 mod types. This is listed in the Skyrim doc.

Here would be a potentially directory structure:

```bash
docs
    mod_types
        mod_types_list.md # Contains a full list of all mod types + definitions
        bg3.md #                 # Contains a list of mods for BG3 specifically.
        cyberpunk.md         # Contains a list of mods for Cyberpunk specifically.
        skyrim.md                # Contains a list of mods for Skyrim specifically.
```

^ We can definitely make this much cleaner if we decide to create an actually docs site that has additional functionality like tabbing, etc. But this is probably the "cleanest" way currently, since these are just plain markdown files in a GitHub repository.

cc @CHollingworth

Reply:
Quick follow up:

I'm more than happy to get something like this set up. Its free to host and already works well with GitHub:

https://docs.github.com/en/pages/setting-up-a-github-pages-site-with-jekyll/about-github-pages-and-jekyll